AmplifyTestCommon/Models/CustomPrimaryKey/CommentWithCompositeKeyUnidirectional.swift

```diff
@@ -0,0 +1,44 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+// swiftlint:disable all
+import Amplify
+import Foundation
+
+public struct CommentWithCompositeKeyUnidirectional: Model {
+  public let id: String
+  public let content: String
+  public var createdAt: Temporal.DateTime?
+  public var updatedAt: Temporal.DateTime?
+  public var postWithCompositeKeyUnidirectionalCommentsId: String?
+  public var postWithCompositeKeyUnidirectionalCommentsTitle: String?
```

Comment:
is this the correct code generated file? should it be operating on the optional association like `public var post: Post?` ?

Reply:
Not in case of unidirectional has-many (@hasMany without a @belongsTo)